web/profiles/joinup/joinup.behat.inc

```diff
@@ -475,4 +475,26 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
     $this->getSession()->getPage()->find('named', ['fieldset', $fieldset])->pressButton($button);
   }
 
+  /**
+   * Fills a date or time field at a datetime widget.
+   *
+   * Example: When I fill in "Date" with "29-08-2016" at "Start date".
+   * Example: When I fill in "Time" with "26:59:00" at "Start date".
+   *
+   * @When I fill in :field with :value at the :field_group date time field
+   */
+  public function fillDateField($field, $value, $field_group) {
+    // From the title, we navigate two parents back to reach the field.
```

Comment:
This is not what we do anymore now, we should explain why we are using array_filter() maybe.